pages/home.md

```diff
@@ -68,15 +76,15 @@ graphic_list:
   <div class="grid-container tablet:margin-top-6">
     <div class="grid-row grid-gap">
       <div class="tablet:grid-col">
-        <img src="{{ site.baseurl }}/img/home/showcase/pubmed.jpg" alt="">
+        <img src="{{ site.baseurl }}/img/webp/home/showcase/pubmed.webp" alt="Preview image of NASA Glenn Research Center home page" width="960" height="540">
```

Comment:
Needs a unique description, currently shows the same as NASA above.

Reply:
Good catch! Updated in [7a40b5a](https://github.com/uswds/uswds-site/pull/1672/commits/7a40b5a8ea0083cc99d37f4eae8f5594fa27b1c4)